test/test_cursor.py

```diff
@@ -1455,6 +1455,14 @@ def test_getMore_does_not_send_readPreference(self):
         self.assertEqual('getMore', started[1].command_name)
         self.assertNotIn('$readPreference', started[1].command)
 
+    @client_context.require_no_mongos
+    def test_exhaust(self):
+        c = self.db.test
+        c.drop()
+        c.insert_many({'_id': i} for i in range(200))
+        result = list(c.find(cursor_type=CursorType.EXHAUST, batch_size=50))
+        self.assertEqual([{'_id': i} for i in range(200)], result)
```

Comment:
Does this test do anything different from TestCollection.test_exhaust?

Reply:
Ping.

Reply:
Ah sorry, yes this test uses a `batch_size` to ensure results are returned in multiple batches. If you'd prefer, I can make the `TestCollection` test do the same thing but I think it makes sense to test this here.

Reply:
This test doesn't actually verify that the batch_size is used, only that all document are returned. I think we should delete this test as a duplicate of the command monitoring test (which does actually verify the command events).

Reply:
Agreed. Removed this test.